rules.py

```diff
@@ -216,31 +219,84 @@ class ElectoralDistrictOcdId(base.BaseRule):
     _OCDID_URL = (
         "https://raw.github.com/opencivicdata/ocd-division-ids/master/identifiers/country-us.csv"
     )
-
+    github_repo = None
+    
     def __init__(self, election_tree, schema_file):
         super(ElectoralDistrictOcdId, self).__init__(election_tree, schema_file)
+        g = Github()
+        self.github_repo = g.get_repo("opencivicdata/ocd-division-ids")
```

Comment:
Might be worth parameterizing some of these values. E.g.,
GITHUB_REPO="opencivicdata/ocd-division-ids"
GITHUB_DIR="identifiers"
GITHUB_FILE="country-us.csv"
GITHUB_BRANCH="master"
_OCDID_URL = ( '/'.join(['https://raw.github.com', GITHUB_REPO, GITHUB_BRANCH, GITHUB_DIR, GITHUB_FILE))